src/pandas_profiling/visualisation/plot.py

```diff
@@ -511,3 +511,27 @@ def plot_acf_pacf(config: Settings, series: pd.Series, figsize: tuple = (15, 5))
     plot_pacf(series.dropna(), lags=lag, ax=axes[1], title="PACF", method="ywm")
 
     return plot_360_n0sc0pe(config)
+
+
+def plot_timeseries_heatmap(
```

Comment:
This is an auxiliary function that does not belong to the ProfileReport, for that reason please consider adding this to be added outside of the methods that are specific to the ProfileReport.